doc/data-retention-policy.md

```diff
@@ -0,0 +1,27 @@
+# Data Retention Policy
+
+We want to prevent the loss of data when there are processing issues, whilst minimising the amount of time we hold any
+data for.
+
+## Maximum Retention Periods
+
+All of our source queues have a maximum retention period of 4 days. This is to allow for any processing issues to be
```

Comment:
For what it's worth I'd be inclined to make this 7 days to account for the sometimes 4-5 days of developer non-availability over some bank holiday periods eg Christmas.

Reply:
Sure - I had a little bit of discussion about this with Ethan and Olly, and we were discussing 4 days vs 7 days, but went with 4 days mostly because that's the number we'd already given to DWP, and that 4 days of developer non-availability is rare. But I think it probably makes sense to go with 7, and we can just tell DWP we've changed it